poseObject: [0.0,0.8,0.05,0.0,0.0,0.0]

radiusObject: 0.1

radiusTrajectory: 0.3

trajectoryStepsNumber: 30